src/theme/palettes/dark.ts

```diff
@@ -62,7 +62,7 @@ const darkPalette: ThemePalette = {
             default: c.grey[900],
             paper: c.grey[700],
             accent: 'rgba(255,255,255,0.06)',
-            main: c.grey[800],
+            main: '#242424',
```

Comment:
shouldn't we rather change a value in the `grey` colours and use it here? 😬 just one that we haven't used elsewhere maybe?